apps/mobile/src/screens/Login/LandingScreen.tsx

```diff
@@ -28,8 +30,17 @@ export function LandingScreen() {
   const navigation = useNavigation<LoginStackNavigatorProp>();
 
   const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
-
+  const [hasSeenOnboardingVideo, setHasSeenOnboardingVideo] = useState(false);
   const [error, setError] = useState('');
+
+  useEffect(() => {
+    AsyncStorage.getItem(SEEN_ONBOARDING_VIDEO_STORAGE_KEY).then((value) => {
```

Comment:
we should be able to use `usePersistedState` , we have this hook in the mobile repo now, works similar to web